.github/changelog-processor.py

```diff
@@ -26,21 +26,38 @@
     help="Should a release be made? Prints `1` or `0`.",
     action="store_true"
 )
+group.add_argument(
+    "--print-changelog-until-previous-version",
+    dest="changelog_until_previous",
+    help="Print the entire changelog until the previous version.",
```

Comment:
It doesn't really make sense to say "until" something in the past, but I see what you mean that the CHANGELOG file itself goes backwards in time. IMO `--print-changelog-from-last-release` expresses this better.

Reply:
My naming is always shit :D